src/pages/home/report/ReportActionContextMenu.js

```diff
@@ -62,6 +65,9 @@ class ReportActionContextMenu extends React.Component {
     constructor(props) {
         super(props);
 
+        this.deleteAccepted = this.deleteAccepted.bind(this);
+        this.deleteCanceled = this.deleteCanceled.bind(this);
```

Comment:
Please make sure all the function-binding happens in the same place. Right now there are these two functions bound here and another lonely one on line 148. Let's move all those together for organization sake.